crates/gpui/src/platform.rs

```diff
@@ -187,6 +187,7 @@ pub(crate) trait PlatformWindow: HasWindowHandle + HasDisplayHandle {
     fn on_appearance_changed(&self, callback: Box<dyn FnMut()>);
     fn is_topmost_for_position(&self, position: Point<Pixels>) -> bool;
     fn draw(&self, scene: &Scene);
+    fn invalidate(&self);
```

Comment:
What is this new `invalidate` method for?

Reply:
`invalidate` is used for `TestWindow`

Reply:
Yeah, I don't think we need this yet.

Reply:
Okay, i didn't know this was used in the mac related code too. i guess it can be removed for now

Reply:
I think the reason we had this was because the old mac rendering code used it which this code was based on, I think the correct fix is just to remove the invalidate method from linux window impl